tests/tools/importexport/migration/conftest.py

```diff
@@ -54,9 +55,9 @@ def _migrate(filename_archive, version_old, version_new, migration_method, archi
         out_path = tmp_path / 'out.aiida'
 
         if zipfile.is_zipfile(archive_path):
-            safe_extract_zip(archive_path, out_path)
+            ZipPath(archive_path).extract_tree(out_path)
```

Comment:
`extract_tree` seems to differ in style from `puttree`

Reply:
again you should raise this in archive-path

Reply:
given that `shutil` also uses [inconsistent](https://docs.python.org/3/library/shutil.html#shutil.copyfileobj) [function](https://docs.python.org/3/library/shutil.html#shutil.get_archive_formats) naming, I'm not going to insist